lib/tasks/stimulus_reflex/install.rake

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 require "fileutils"
+require_relative "../../stimulus_reflex/version"
```

Comment:
@piotrwodz I think you should change this to `require "stimulus_reflex/version"`, a relative path might mess things up. Not sure, but it's probably better that way.

Reply:
I'm going to do this locally before I cut the gem.